portal/templates/redesign/teach_new/onboarding_school.html

```diff
@@ -71,7 +78,48 @@ <h4 class="col-sm-offset-2">Everything you need to start coding with your class
         </div>
     </div>
 </div>
+{% if teacher.pending_join_request %}
+<div id="onboarding_revoke_request_page"></div>
+
+<div class="sub-nav sub-nav--warning">
+    <div class="row">
+        <img src='{% static "portal/img_new/warning.png" %}'>
+        <p class="body-text"><i>Your request to join {{ teacher.pending_join_request.name }}, {{ teacher.pending_join_request.postcode }} has been submitted</i></p>
```

Comment:
Postcode has been capslocked in organisation_new.py upon creation of a school :)